scripts/spack/packages/axom/package.py

```diff
@@ -244,6 +260,71 @@ def initconfig_hardware_entries(self):
             entries.append(
                 cmake_cache_option("gtest_disable_pthreads", True))
 
+        if "+rocm" in spec:
+            entries.append("#------------------{0}\n".format("-" * 60))
+            entries.append("# HIP\n")
+            entries.append("#------------------{0}\n\n".format("-" * 60))
+
+            entries.append(cmake_cache_option("ENABLE_HIP", True))
+
+            hip_root = spec['hip'].prefix
+            rocm_root = hip_root + "/.."
+
+            entries.append(cmake_cache_string("HIP_ROOT_DIR",
+                                        hip_root))
+            entries.append(cmake_cache_string("HIP_CLANG_PATH",
+                                        rocm_root + '/llvm/bin'))
+            archs = self.spec.variants['amdgpu_target'].value
+            if archs != 'none':
+                arch_str = ",".join(archs)
+                entries.append(cmake_cache_string(
+                    "HIP_HIPCC_FLAGS", '--amdgpu-target={0}'.format(arch_str)))
+                entries.append(cmake_cache_string(
+                    "CMAKE_HIP_ARCHITECTURES", arch_str))
+            entries.append(cmake_cache_string("HIP_RUNTIME_INCLUDE_DIRS",
```

Comment:
Does all this go away with the new BLT hip support?

Reply:
Most of these flags are unnecessary for current and new blt, I kept CMAKE_HIP_ARCHITECTURES  and HIP_ROOT_DIR. I had copied these over from [RAJA's spack recipe](https://github.com/LLNL/RAJA/blob/8e8cc96cbccda2bfc33b14b57a8591b2cf9ca342/scripts/spack_packages/raja/package.py#L296)